org-super-agenda.el

```diff
@@ -238,9 +242,16 @@ If ANY is non-nil, return as soon as FORM returns non-nil."
   "Return list of tags in agenda item string S."
   (org-find-text-property-in-string 'tags s))
 
+(defun org-super-agenda--make-agenda-header-separator ()
+  "Return a string made from `org-super-agenda-header-separator'."
+  (let ((s org-super-agenda-header-separator))
+    (typecase s
```

Comment:
This should use `cl-typecase`.

Reply:
Could you please elaborate?

Reply:
The `typecase` function is an alias for `cl-typecase`, from `cl-lib.el`.  If you use plain `typecase`, Emacs will issue a byte-compile warning.  One of the idiosyncrasies of Emacs (that change was made around Emacs 24.3 IIRC).

Reply:
Thank you. I use Common Lisp most of the time so I did not realize that this was just an alias.

New lesson learned!